meerk40t/gui/scene/scene.py

```diff
@@ -14,6 +14,9 @@
 from meerk40t.kernel import Job, Module
 from meerk40t.svgelements import Matrix, Point, Viewbox
 
+# Comment this line out as soon as PR805 gets merged
+# from meerk40t.gui.wxutils import get_key_name
+
```

Comment:
PR #801 was merged. #805 is a github issue template thing

Reply:
The use of get_key_name there is thus a bit redundant as code or whatever. Might take merge from main

Reply:
As indicated in the comments, these were just proxy routines to be replaced those in the recent PR from Sophist
@Sophist-UK : I will have a look at the code in wxmain to think about the key-up event. There are some challenges around i.e. Ctrl-Shift pressed simultaneously and then released... I can easily work around this in my code, but I think it deserves a 'real solution'